### ISSUE TITLE
Getting Cannot read property 'invalid' of undefined

### ISSUE THREAD

**Issue by notdrone** [mike-works/pwa-fundamentals#169]
**System Information**
OS type: macOS
Node version: v8.11.3

**To Reproduce**
1. checkout master
2. run `npm i`
3. run `npm run prepcerts`
4. run `npm run wathc:ui`
5. Here's the stack trace:
```
pwa-fundamentals/node_modules/webpack-dev-middleware/lib/context.js:95
  context.compiler.hooks.invalid.tap('WebpackDevMiddleware', invalid);
                         ^

TypeError: Cannot read property 'invalid' of undefined
    at ctx (/Users/notdrone/Documents/fem/pwa/pwa-fundamentals/node_modules/webpack-dev-middleware/lib/context.js:95:26)
    at wdm (/Users/notdrone/Documents/fem/pwa/pwa-fundamentals/node_modules/webpack-dev-middleware/index.js:46:19)
    at Object.<anonymous> (/Users/notdrone/Documents/fem/pwa/pwa-fundamentals/server/asset-server/index.js:72:27)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by mike-north** [comment]
I can reproduce this and am working on it today. Almost certainly has to do with recent webpack/webpack-dev-server upgrades

**Comment by cortesben** [comment]
I am watching the `Progressive Web Applications and Offline` and wanted to follow along but I am getting the same error on this issue.

```shell
lib/context.js:95
[UI]   context.compiler.hooks.invalid.tap('WebpackDevMiddleware', invalid);
```

**Comment by mike-north** [comment]
Should be fixed on master ( bb72c7b ) now.

**Comment by cortesben** [comment]
Yup Thank you!
